src/Microsoft.Identity.Web.OidcFIC/OidcFicSignedAssertionProviderExtensions.cs

```diff
@@ -19,7 +20,11 @@ public static class OidcFicSignedAssertionProviderExtensions
         /// <returns>the service collection for chaining.</returns>
         public static IServiceCollection AddOidcFic(this IServiceCollection services)
         {
-            services.AddSingleton<ICustomSignedAssertionProvider, OidcIdpSignedAssertionLoader>();
+            services.TryAddEnumerable(
+                new[]
+                {
+                ServiceDescriptor.Singleton<ICustomSignedAssertionProvider, OidcIdpSignedAssertionLoader>()
+                });
```

Comment:
```suggestion
            services.TryAddEnumerable(ServiceDescriptor.Singleton<ICustomSignedAssertionProvider, OidcIdpSignedAssertionLoader>());
```

This overload of TryAddEnumerable that takes an IEnumerable of ServiceDescriptor only calls the other overload on the individual items. So we can just use the method that takes a single item. Also there is a more simple syntax for creating ServiceDescriptor that I found recently.